src/libs/antares/study/parts/hydro/container.h

```diff
@@ -171,6 +171,8 @@ class PartHydro
     PreproHydro* prepro;
     //! Data for time-series
     DataSeriesHydro* series;
+    //! difference between initial and final reservoir levels for all scenario-builder years
+    std::vector<double> finalReservoirLevelCorrection;
```

Comment:
Currently, this vector is written to, but it isn't used. Is it a mistake or work-in-progress ?

Reply:
WIP. See new commits.